src/Types.ts

```diff
@@ -795,6 +795,11 @@ export interface IAccountInformation {
      */
     total_frozen: JSBI;
 
+    /**
+     * Total frozen amount of account
```

Comment:
The document is invalid. (copy and paste)

Reply:
resolved